includes/compat/trait-wc-stripe-subscriptions.php

```diff
@@ -1161,8 +1163,9 @@ public function redirect_after_early_renewal( $url ) {
 	 */
 	protected function maybe_process_subscription_early_renewal_success( $order, $intent ) {
 		if ( WC_Stripe_Subscriptions_Helper::is_subscriptions_enabled() && isset( $_GET['early_renewal'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
-			if ( function_exists( 'wcs_update_dates_after_early_renewal' ) && function_exists( 'wcs_get_subscription' ) ) {
-				wcs_update_dates_after_early_renewal( wcs_get_subscription( $order->get_meta( '_subscription_renewal' ) ), $order );
+			$subscription = WC_Stripe_Subscription::get_by_id( $order->get_meta( '_subscription_renewal' ) );
+			if ( function_exists( 'wcs_update_dates_after_early_renewal' ) && $subscription ) {
+				wcs_update_dates_after_early_renewal( $subscription, $order );
```

Comment:
Another case where I think we need to pass the `WC_Subscription` instance to the function.
```suggestion
				wcs_update_dates_after_early_renewal( $subscription->get_wc_subscription(), $order );
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4254/commits/415dcb68ed8d61f6b6f06b2f89ef5579d470cc49